jscomp/bsb/bsb_pkg.ml

```diff
@@ -106,6 +106,22 @@ let pkg_name_as_variable package =
   |> fun s -> Ext_string.split s '-'
   |> String.concat "_"
 
+let rec find_dep_path ic package_name =
```

Comment:
One thing you could do that might make this function simpler is make this function do the try/catch and return an option of string

Reply:
Good idea!